==================================== INPUT ====================================
# expected-to-throw

fragment refetchableWithoutNameConflictWithOperation on Node
  @refetchable
{
  __typename
}

query refetchableWithoutNameConflictWithOperationRefetchQuery {
  node(id: "y") {
    __typename
  }
}
==================================== ERROR ====================================
✖︎ A unique query name has to be specified in `@refetchable`, an operation `refetchableWithoutNameConflictWithOperationRefetchQuery` already exists.

  refetchable_without_name_conflict_with_operation.invalid.graphql:4:3
    3 │ fragment refetchableWithoutNameConflictWithOperation on Node
    4 │   @refetchable
      │   ^^^^^^^^^^^^
    5 │ {

  ℹ︎ an operation with that name is already defined here

  refetchable_without_name_conflict_with_operation.invalid.graphql:9:7
    8 │ 
    9 │ query refetchableWithoutNameConflictWithOperationRefetchQuery {
      │       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   10 │   node(id: "y") {
